<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>&lt;aumuell@reserv.at&gt;</author>
 <class>OrganizeCollectionDialogBase</class>
 <widget class="QWidget" name="OrganizeCollectionDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>593</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Organize Files</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="_3">
     <property name="margin">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="folderLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Base directory under which to put files</string>
       </property>
       <property name="text">
        <string>C&amp;ollection Folder:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>folderCombo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KComboBox" name="folderCombo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Character Replacement</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="spaceCheck">
        <property name="toolTip">
         <string>If checked, convert spaces to underscores.</string>
        </property>
        <property name="whatsThis">
         <string>If checked all spaces will be replaced with spaces in the file names. If not checked spaces will be left as they are.</string>
        </property>
        <property name="text">
         <string>Replace spaces with &amp;underscores</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ignoreTheCheck">
        <property name="toolTip">
         <string>If checked, postfix artists' names starting with 'The' with ', The'.</string>
        </property>
        <property name="whatsThis">
         <string>If checked all artists that have the word &quot;The&quot; at the beginning will be treated as if the &quot;The&quot; was not there. Example, &quot;The Beatles&quot; would become &quot;Beatles, The&quot;</string>
        </property>
        <property name="text">
         <string>I&amp;gnore 'The' in artist names</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="asciiCheck">
        <property name="toolTip">
         <string>If checked all non-ASCII characters will be replaced with underscores. Do not check this option if you have non-english characters in your tags. Most people can leave this option unchecked.</string>
        </property>
        <property name="whatsThis">
         <string>If checked all non-ASCII characters will be replaced with underscores. Do not check this option if you have non-english characters in your tags. Most people can leave this option unchecked.</string>
        </property>
        <property name="text">
         <string>Restrict to &amp;ASCII</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="vfatCheck">
        <property name="toolTip">
         <string>If checked, replace characters that are incompatible with MS-DOS/VFAT file systems with underscores. Incompatible characters include |?*&lt;&quot;:&gt;+[] and unicode control characters. Most people can leave this option enabled as it ensures copying to media devices will function correctly.</string>
        </property>
        <property name="whatsThis">
         <string>If checked, replace characters that are incompatible with MS-DOS/VFAT file systems with underscores. Incompatible characters include |?*&lt;&quot;:&gt;+[] and unicode control characters. Most people can leave this option enabled as it ensures copying to media devices will function correctly.</string>
        </property>
        <property name="text">
         <string>&amp;VFAT safe names</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>47</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_2">
        <item>
         <widget class="QLabel" name="textLabel1">
          <property name="text">
           <string>Replace</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KLineEdit" name="regexpEdit">
          <property name="toolTip">
           <string>Regular expression</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel2">
          <property name="text">
           <string>with</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KLineEdit" name="replaceEdit">
          <property name="toolTip">
           <string>Character string</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="previewBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Destination Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTableWidget" name="previewTableWidget">
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Original</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Preview</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="previewText">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>This is what the file names will look like after renaming.</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="conflictLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <item>
         <spacer name="spacer2_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>347</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="overwriteCheck">
          <property name="toolTip">
           <string>If checked, overwrite files of the same name without asking.</string>
          </property>
          <property name="whatsThis">
           <string>Check this to overwrite files of the same name automatically. (Warning: Overwriting is not undoable)</string>
          </property>
          <property name="text">
           <string>Overwrite &amp;Destination</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="0"/>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">kcombobox.h</include>
  <include location="local">klineedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
